scimono-compliance-tests/Readme.md

```diff
@@ -43,6 +43,11 @@ Values for the following command line arguments: `oauth.clientId`;  `oauth.secre
 java -jar scimono-compliance-tests-${version}-shaded.jar scim.service.url=http://localhost:8080/scim/v2 auth.type=Oauth oauth.grant=client_credentials oauth.clientId=admin oauth.secret=secret oauth.service.url=http://localhost:8080/oauth/token
 ```
 
+### Custom Headers
+```
+java -jar scimono-compliance-tests-${version}-shaded.jar scim.service.url=http://localhost:8080/scim/v2 auth.type=Oauth oauth.grant=client_credentials oauth.clientId=admin oauth.secret=secret oauth.service.url=http://localhost:8080/oauth/token custom.headers='key:v1;v2','key2:value'
+```
```

Comment:
Please describe that you expect the headers to be in singe quotes

Reply:
Also, at the bottom of the page, there is a list with all accepted parameters. Please, populate it.